plasticos_web_leads/models/web_lead.py

```
@@ -0,0 +1,427 @@
+        """Retry processing a lead that errored."""
+        for rec in self:
+            if rec.state != "error":
+                raise UserError("Only errored leads can be retried.")
+            if rec.decision == "hot":
+                rec._process_hot_lead()
+
+    def action_force_create_intake(self):
+        """Force-create an intake from a COLD lead (manual override)."""
+        for rec in self:
+            if rec.intake_id:
+                raise UserError("Intake already exists for this lead.")
```

Ruff (0.15.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)